README.md

```diff
@@ -7,6 +7,8 @@ _vswhere_ is designed to be a redistributable, single-file executable that can b
 
 You can emit different formats for information based on what your scripts can consume, including plain text, JSON, and XML. Pull requests may be accepted for other common formats as well.
 
+_vswhere_ is included with the installer as of Visual Studio 2017 version 15.2 and later, and can be found at the following location: `%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe`. The binary may be copied from that location as needed, [installed from Chocolatey](https://chocolatey.org/packages/vswhere), or the latest version may be compiled from the source code in this repo.
```

Comment:
1. "installed **using** Chocolatey" would be better.
2. Rather than recommending compiling from source (certainly doable, but overkill for most) they should be directed to the https://github.com/Microsoft/vswhere/releases page to download.
3. It's also available using nuget (same package as chocolatey, but different repo), which is handy for build scripts. I actually use it that way myself in this and other projects. With that said, maybe we should actually direct them to https://github.com/Microsoft/vswhere/wiki/Installing with all that information already.